web-client/src/views/StartCase/StartCaseStep5.tsx

```diff
@@ -146,23 +146,23 @@ export const StartCaseStep5 = connect(
                           </div>
                         </div>
 
-                        {form.ownershipDisclosureFile && (
+                        {form.corporateDisclosureFile && (
                           <div className="margin-top-3 margin-bottom-3">
                             <span
                               className="usa-label usa-label-display margin-top-3"
                               htmlFor="filing-parties"
                             >
-                              Ownership Disclosure Statement
+                              Corporate Disclosure Statement
                             </span>
                             <div>
                               <div className="grid-row">
                                 <div className="grid-col flex-auto">
                                   <PDFPreviewButton
-                                    file={form.ownershipDisclosureFile}
-                                    id="ods-preview-button"
+                                    file={form.corporateDisclosureFile}
+                                    id="cds-preview-button"
                                     shouldAbbreviateTitle={false}
                                     shouldWrapText={true}
-                                    title="Ownership Disclosure Statement"
+                                    title="corporate Disclosure Statement"
```

Comment:
Nit: Capitalization

Reply:
![Eagle eye](https://media.giphy.com/media/qVVVfmHDMBZug/giphy.gif)

Thank you! You win this week's eagle eye award